src/bootstrap/compile.rs

```diff
@@ -231,6 +231,16 @@ pub fn std_cargo(builder: &Builder<'_>, target: Interned<String>, cargo: &mut Ca
             }
         }
     }
+
+    // libstd must be built with bitcode so that the produced rlibs can be used
+    // for both LTO builds (which use bitcode) and non-LTO builds (which use
+    // object code).
+    //
+    // But we don't bother for the stage 0 compiler because it's never used
+    // with LTO.
+    if stage >= 1 {
+        cargo.rustflag("-Cbitcode-in-rlib=yes");
```

Comment:
Somewhat confusing to see this explicitly set to the default value?

Reply:
Oh this overwrite cargo which changes the default?

Reply:
True, I will clarify the comment to explain that Cargo overrides the default, and we have to override Cargo's override.